bower.json

```diff
@@ -12,31 +12,5 @@
     "type": "git",
     "url": "https://github.com/cfpb/capital-framework.git"
   },
-  "license": "CC0",
-  "dependencies": {
-    "capital-framework": "~1.0.0"
-  },
-  "exportsOverride": {
-    "cf-*": {
-      "css": "src/**/*.css",
-      "less": "src/**/*.less",
-      "fonts": "src/fonts/*.*",
-      "js": "src/**/*.js"
-    },
-    "box-sizing-polyfill": {
-      "js": "boxsizing.htc"
-    },
-    "html5shiv": {
-      "js": "dist/html5shiv-printshiv.min.js"
-    },
-    "jquery.easing": {
-      "js": "js/jquery.easing.js"
-    }
-  },
-  "resolutions": {
-    "cf-expandables": "^1.0.0",
-    "cf-layout": "^1.0.0",
-    "cf-pagination": "^1.0.0",
-    "cf-typography": "^1.0.0"
-  }
+  "license": "CC0"
```

Comment:
Sounds good. I left it in mostly because I wasn't 100% sure it wasn't needed anymore, but now I am. The gruntfile uses this manifest to build banners, but I'll swap this out for `package.json` there and ditch this one.